src/Cassandra/RowPopulators/Row.cs

```diff
@@ -133,12 +133,9 @@ public virtual bool IsNull(int index)
         /// </summary>
         public CqlColumn GetColumn(string name)
         {
-            int index;
-            if (!ColumnIndexes.TryGetValue(name, out index))
-            {
-                return null;
-            }
-            return Columns[index];
+            return !ColumnIndexes.TryGetValue(name, out var index) 
+                ? null 
+                : Columns[index];
```

Comment:
```suggestion
            return ColumnIndexes.TryGetValue(name, out var index) 
                ? Columns[index]
                : null;
```

easier to read imo

Reply:
Yes of course, I'm still analyzing the whole code, but I thought about using the pareto principle here!

I will make the adjustment!